tests/integration/common/s2n_test_openssl.py

```diff
@@ -77,6 +77,6 @@ def get_openssl(scenario):
     return openssl
 
 
-def run_openssl_connection_test(scenarios, test_func=None):
-    return util.run_connection_test(get_openssl, scenarios, test_func)
+def run_openssl_connection_test(scenarios, **kwargs):
```

Comment:
non-blocking nit, this pattern hides arguments you can pass to the connection test runner. This isn't an issue here, but I've seen codebases where kwargs replaces what should a defined interface. Without structure, different objects start interpreting the same named argument differently. Its just something to keep in mind as we go.

Reply:
Definitely. I agree **kwargs shouldn't be used for any arguments that run_openssl_connection_test will actually use, just for arguments that it will blindly pass on. My goal here is to preserve the defaults defined by run_connection_test, the only function that cares about those args.